packages/account/src/Sections/Verification/ProofOfOwnership/__test__/ExpandedCard.spec.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ExpandedCard from '../ExpandedCard.jsx';
+
+const formatIdentifier = (id, type) => {
+    let formattedID = id;
+    if (type === 'IcCreditCard' || type === 'IcStockVisa' || type === 'IcStockMasterCard')
+        formattedID = `${id.substr(0, 6)}XXXXXX${id.substr(12)}`;
+    else if (type === 'IcEwallet') return formattedID;
+    return formattedID
+        .replace(/\s/g, '')
+        .replace(/(\w{4})/g, '$1 ')
+        .trim();
+};
+
+describe('ExpandedCard.jsx', () => {
+    let cardDetails;
+
+    beforeAll(() => {
+        cardDetails = {
+            icon: 'IcCreditCard',
+            creation_time: '9 o clock',
+            id: 'abc12340',
+            payment_method: 'credit_debit_card',
+            payment_method_identifier: 'credit_debit_card',
+            title: 'Credit / Debit card',
+            paragraphs: [
+                'Upload a photo of your card or bank statement showing your name and card number. Your card number must only show the first 6 and last 4 digits.',
+            ],
+            input_label: 'Card Number',
+        };
+    });
+    it('should display correct identifier', () => {
+        render(<ExpandedCard cardDetails={cardDetails} identifier={cardDetails.payment_method_identifier} />);
+        const element = screen.getByText('Card Number', { exact: true });
+        expect(element).toBeInTheDocument();
+    });
+    it('should show example link for credit/debit card', async () => {
```

Comment:
just out of curiosity, why did you use async here?

Reply:
Hello @iman-fs,

I just forgot to remove that. At some point, I used waitFor and I changed my implementation. But I forgo that in.

Thank you